exporter/collector/metricsexporter.go

```diff
@@ -448,9 +450,17 @@ func (m *metricMapper) summaryPointToTimeSeries(
 	quantiles := point.QuantileValues()
 	for i := 0; i < quantiles.Len(); i++ {
 		quantile := quantiles.At(i)
-		pLabel := labels{
-			// Convert to percentile.
-			"percentile": fmt.Sprintf("%f", quantile.Quantile()*100),
+		var pLabel labels
+		if m.cfg.MetricConfig.SummaryPercentiles {
+			pLabel = labels{
+				// Convert to percentile.
+				"percentile": fmt.Sprintf("%f", quantile.Quantile()*100),
+			}
+		} else {
+			pLabel = labels{
+				// Convert to quantile.
+				"quantile": fmt.Sprintf("%f", quantile.Quantile()),
+			}
```

Comment:
Nit: This function is pretty long already, would you consider extracting this logic out into a small utility?

Reply:
Done